### PR TITLE
GA Subworkflow Nodes

### PR DESCRIPTION
**Pull request by dvargas92495**

### CODE REVIEW

File: fern/docs/content/help/workflows/experimenting.mdx [hunk 1]
```diff
@@ -9,52 +9,95 @@ Workflows is a product in Vellum's LLM dev platform that helps you quickly proto
 
 The Workflows UI consists of a graphical app builder where you can string together various nodes and test various input values through this system. Each prompt can also be tested extensively through Playground & Test Suites. When implemented effectively, Workflows can help you build advanced LLM applications
 
-## Supported Node Types, as of August 14, 2023
```
Comment (dvargas92495) [inline fern/docs/content/help/workflows/experimenting.mdx:12]: I posit that:
- This should be a page-level feature that fern supports
- It encourages docs-drift that we should avoid

File: fern/docs/content/help/workflows/experimenting.mdx [hunk 2]
```diff
@@ -9,52 +9,95 @@ Workflows is a product in Vellum's LLM dev platform that helps you quickly proto
...
+## Supported Node Types
 
 The names of nodes can be edited in-line once the node has been added to the UI. You can add as many nodes as you want to your Workflow. Since there is extreme flexibility in how Workflows can be created, you can modify your architecture to your desire by starting from these building blocks. Common architectures are shared in a separate help center doc ([Common Workflow Architectures](/help-center/documents/uploading-documents)) and will be updated over time.
 
 ![Workflow Nodes](https://storage.googleapis.com/vellum-public/help-docs/workflow_nodes.png)
 
 Drag and drop a node from the side panel into the UI and start building
 
-### 1. Prompt Nodes
+### Prompt Nodes
```
Comment (dvargas92495) [inline fern/docs/content/help/workflows/experimenting.mdx:20]: number ordering in headers is brittle to changes, breaking direct links

File: fern/docs/content/change-log/2024-03.mdx [hunk 3]
```diff
@@ -2,6 +2,18 @@
 title: Changelog | March, 2024
 ---
 
+## Subworkflow Nodes
+
+_March 14th, 2024_
+
+Are your Workflows becoming giant and unwieldy? Wish you could define composable groups of nodes to be used across Workflows?
+
+We're excited to introduce the latest node type in the Workflows node picker - Subworkflow Nodes! With Subworkflow Nodes, you could link directly to deployed Workflows to reuse commonly grouped nodes and execution logic. Subworkflow Nodes also supports release tagging, giving users the flexibility to either pin to a specific version (say, `production`) or always automatically update with `LATEST`.
```
Comment (nihalathanikkal) [inline fern/docs/content/change-log/2024-03.mdx:11]: > With Subworkflow Nodes, you could link directly to deployed Workflows to reuse commonly grouped nodes and execution logic. 

 [nit] you **_can now_** link

File: fern/docs/content/help/workflows/experimenting.mdx [hunk 4]
```diff
@@ -9,52 +9,95 @@ Workflows is a product in Vellum's LLM dev platform that helps you quickly proto
...
 
 Upon creating a Prompt Node you’ll be asked to import a prompt from an existing Deployment, Sandbox, or create one from scratch. Prompts are defined by their variables, prompt template, model provider, and parameters. Refer to this help center article to learn more about our prompt syntax ([Vellum Prompt Template Syntax](/help-center/prompts/prompt-engineering)).
 
 ![Prompt Node](https://storage.googleapis.com/vellum-public/help-docs/prompt_node.png)
 
-### 2. Search Node
+
+### Templating Nodes
+
+The templating node allows you to perform custom data transformations on a set of defined inputs into a new output. You could use this to define constants, manipulate data before feeding into a prompt, or massage a response to a format to your liking.
```
Comment (nihalathanikkal) [inline fern/docs/content/help/workflows/experimenting.mdx:31]: [nit] "Templating Node" for consistency
[nit] You could use this -> You **_can_** use this
[nit] rephrase

The Templating Node allows you to perform custom data transformations on a set of defined inputs to create new outputs.

File: fern/docs/content/help/workflows/experimenting.mdx [hunk 5]
```diff
@@ -9,52 +9,95 @@ Workflows is a product in Vellum's LLM dev platform that helps you quickly proto
...
 The Search Node returns results from a Document Index stored inside Vellum Search. Once your documents are uploaded in an index (details on how to do that here: [Uploading Documents](/help-center/documents/uploading-documents)), you can start using them in a Workflow.
 
 The index in a Search Node can be fixed for the Workflow or chosen dynamically based on the output of an upstream node. Additional configuration options, similar to the ones in Vellum Search are also available in the Search Node.
 
 ![Search Node](https://storage.googleapis.com/vellum-public/help-docs/search_node.png)
 
-### 3. Conditional Node
+### API Node
+
+The API node invokes an API endpoint and returns back the status code, raw output, and JSON output if applicable. These APIs could be either publically accessible or ones that are privately defined within your backend through the help of Authorization headers and Workspace Secrets. Simply define a URL, HTTP Method, and the body that you would like to send to the desired endpoint.
```
Comment (nihalathanikkal) [inline fern/docs/content/help/workflows/experimenting.mdx:47]: [nit] rephrase for grammar, spelling & consistency:

The API Node invokes an API endpoint and returns back the status code, raw output, and JSON output if applicable. These APIs can be either publicly accessible or privately defined within your backend through the help of Authorization headers and Workspace Secrets. Simply define a URL, HTTP Method, and the body that you would like to send to the desired endpoint.

File: fern/docs/content/help/workflows/experimenting.mdx [hunk 6]
```diff
@@ -9,52 +9,95 @@ Workflows is a product in Vellum's LLM dev platform that helps you quickly proto
...
+
+![API Node](https://storage.googleapis.com/vellum-public/help-docs/api-node.png)
+
+### Code Execution Nodes
+
+The code execution node empowers you to include custom logic defined directly in the workflow. You could even import custom public packages within the node's logic. We support the following languages:
```
Comment (nihalathanikkal) [inline fern/docs/content/help/workflows/experimenting.mdx:53]: [nit] Code Execution Node
[nit] could -> **_can_**

File: fern/docs/content/help/workflows/experimenting.mdx [hunk 7]
```diff
@@ -9,52 +9,95 @@ Workflows is a product in Vellum's LLM dev platform that helps you quickly proto
...
+- Python
+- TypeScript
+
+![Code Execution Node](https://storage.googleapis.com/vellum-public/help-docs/code-exec-node.png)
+
+### Subworkflow Nodes
+
+Subworkflow Nodes are essential for managing giant, complex workflows. Define reusable groups of workflow nodes in one sandbox and have it directly accessible upon deployment from any other workflow in your workspace. Subworkflow nodes also support release tag specification, allowing you the option to always invoke the latest workflow, or pinning to a specific release tag defined by you.
```
Comment (nihalathanikkal) [inline fern/docs/content/help/workflows/experimenting.mdx:61]: [nit]  rephrase: **_Define reusable groups of nodes in one Workflow Sandbox and have them_**

File: fern/docs/content/help/workflows/experimenting.mdx [hunk 8]
```diff
@@ -9,52 +9,95 @@ Workflows is a product in Vellum's LLM dev platform that helps you quickly proto
...
 
 The Final Output Node represents the end of your workflow. Your workflow may have multiple Final Output Nodes if the execution has been branched off from an upstream node.
 
 A name for the output and an output type must be configured here because the response streamed back from the endpoint (when the workflow is taken to production) has this information included.
 
 ![Final Output Node](https://storage.googleapis.com/vellum-public/help-docs/terminal_node.png)
 
+### Error Node
+
+The Error Node enables you to reject the full workflow at any point whenever it gets invoked. There are two types of errors you could raise with this node:
```
Comment (nihalathanikkal) [inline fern/docs/content/help/workflows/experimenting.mdx:95]: [nit] `at any point whenever it gets invoked`
 
 A little vague, maybe can provide some details of when we might expect it to get invoked

**Review by nihalathanikkal — APPROVED** [review]
Looks good! 🚢  Just have some minor nits

File: fern/docs/content/help/workflows/experimenting.mdx (hunk 4 above)
Comment (dvargas92495) [reply fern/docs/content/help/workflows/experimenting.mdx:31]: Taking the rephrase, but making it a singular output (templating nodes only emit 1)

**Pull request merged**